src/layout/Spacer/Spacer.tsx

```diff
@@ -8,18 +8,48 @@ export interface SpacerProps extends React.HTMLAttributes<HTMLDivElement> {
    */
   dense?: boolean;
 
+  /**
+   * If true, the spacer will be inline.
+   */
+  inline?: boolean;
+
   /**
    * The direction of the spacer.
    */
   direction?: 'row' | 'column';
+
+  /**
+   * How the content is arranged inside the spacer.
+   */
+  alignItems?: 'start' | 'end' | 'center';
+
+  /**
+   * How the content is arranged inside the spacer.
+   */
+  justifyContent?: 'start' | 'end' | 'center' | 'spaceBetween';
 }
 
 export const Spacer = React.forwardRef<HTMLDivElement, SpacerProps>(
```

Comment:
This feels more like a 'row' or something now. MUI calls it stack which im not opposed to either - https://mui.com/joy-ui/react-stack/. We could just alias this to a new name so we don't break old code.